app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/DeclarationXml.scala

```diff
@@ -112,7 +113,7 @@ class DeclarationXml {
           {maybeParty("Importer", dec.parties.importer)}
           {dec.documentationAndReferences.headerPreviousDocuments.map(_.toXml)}
           {maybeParty("Seller", dec.parties.seller)}
-          {maybeTradeTerms(dec)}
+          {dec.goodsShipment.tradeTerms.map(_.toXml).getOrElse(NodeSeq.Empty)}
```

Comment:
think this can just be `dec.goodsShipment.tradeTerms.toXml`

Reply:
Nice. Will do